docs/ingredients/accounts.md

```diff
@@ -0,0 +1,41 @@
+- Only an owner program can modify an Accounts state
+- Account owners (i.e Wallets) can sign transactions by providing a signature
+- Signatures are used to verify authenticity, ownership, and approval
+- Verifying the owner of an account and program is important for security
+- Accounts can hold meta data (i.e authority, name, description, price)
+- An authority can be checked on an account to authorize actions inside a program
+- Accounts are charged rent based on space used
+- Accounts can be made rent exempt by supplying a minimum rent balance
+- Accounts are passed into programs using Transactions
+- Programs are actually Accounts marked as executable when deployed
+- Transactions in addition to Instructions and Signatures include Accounts
+- Accounts are marked as read-write, or read-only when including them in transactions
+- Marking accounts as read-only can enable parallel account processing
+- Account owners can provide signatures by signing a transaction
+- Another type of account that can be used by a Program is a PDA
+- A PDA (Program derived Account) gives a program a way provide a signature
+
+#### Delivery
+Accounts store state (data and lamports). Lamports are fractional units of native SOL (0.000000001). While programs themselves do not store state directly, accounts can be used to store all the state a program requires. How does this work? Accounts when setup are assigned an owner. The owner is a program that has exclusive control over any changes to state. The owner program is referenced using a program id (address, or publickey). Anyone can read an account, but only the program that owns the account has the authority to change or permit modifications to the account data, including any debits or transfers from an account balance. If an account is able to recieve SOL or Tokens, anyone can transfer or send a balance to the account. If an account is not owned by a program, the program is only permitted to read its data and credit the account. Why might an account not be able to recieve SOL? 
+
+Accounts can be marked as executable. If the account is marked as executable it is treated as a program. Once marked executable the program data is immutable This is where a comparison to a file has been mentioned, as there appear to be some similarities to a file on your operating system. If a file is marked as executable, it can be loaded and executed by a program (in this case BPF Loader). If an account is not marked as executable, it is treated as a non executable account that can be read and updated based on permissions (authority and ownership), like a file. Accounts are marked executable during a successful program deployment by none other than the BPF Loader. The BPF loader is a native program of Solana and is in charge of deployments, upgrades, and executing programs on chain. This means the BPF Loader owns your program, and the program can own accounts in order to store any state it needs. Cool? 
```

Comment:
Mentioning the an executable account is a program is good, but I am not sure if this goes too in depth on programs and should just link back to the program ingredient instead. Also, I believe program accounts and be mutable and executable and that the original docs are outdated. The UpgradableBPFLoader allows this to happen.

Reply:
it appears I edited some language out that talked about the final key word where it is marked as a final deploy but it is a little more in depth. it depends how encompassing we want core concepts to be. because they are core concepts they may include a lot and go more in depth. The idea that  program can be marked final, or that it is upgradable, and that programs are marked executable and loaded by the BPF loader at runtime, is not something an ingredient may even show, as its part of the run time under normal circumstances and depends on context. Ideally I would want core concepts to convey most of the general and some not so common knowledge. Would prefer not just have core concepts as just high level overviews and scatter related details all over the cookbook with no real way to search or learn about them without a special circumstance. Maybe there is a better way to show related items. If I have a basic question about a program, in regards to how it works, or what its capable of, it should be answered here, If there are more details or deep dives, diagrams, etc, they can be linked somehow as well.

Reply:
Things that we consider common knowledge may not be few a new Solana dev. We ideally want to be able to give anyone the cookbook and have them be able to get started. 

There should be more structure of what information goes where. If we have a bunch of program info in accounts, then not much in the program core concept, we'll run into issues. Readability is key.